deepmd/pd/utils/env_mat_stat.py

```
@@ -0,0 +1,235 @@
+            dtype=env.GLOBAL_PD_FLOAT_PRECISION,
+        ).to(env.DEVICE)
+        if self.last_dim == 4:
+            radial_only = False
+        elif self.last_dim == 1:
+            radial_only = True
+        else:
+            raise ValueError(
+                "last_dim should be 1 for raial-only or 4 for full descriptor."
+            )
+        for system in data:
+            coord, atype, box, natoms = (
```

Ruff

Local variable `natoms` is assigned to but never used

Remove assignment to unused variable `natoms`

(F841)